ruins/core/data_manager.py

```diff
@@ -151,13 +151,23 @@ def __init__(self, datapath: str = None, cache: bool = True, hot_load = False, d
         """
         You can pass in a Config as kwargs.
         """
-        # check if the no config - or config without datapath - was passed
+        # check if no config - or config without datapath - was passed
         if datapath is None:
             from ruins.core import Config
             self.from_config(**Config(**kwargs))
         else:
             self.from_config(datapath=datapath, cache=cache, hot_load=hot_load, debug=debug, **kwargs)
     
+    def read(self, name_or_file: str):
+        # if config init then there is an attrribute called datafile_names
+        if 'datafile_names' in self._config:
+            # if name_or_file exists in Config.datafile_names, the filename stored there is used. Otherwise name_or_file is considered the filename.
+            identifier = self._config['datafile_names'].get(name_or_file, name_or_file) # unittest
+        else:
+            identifier = name_or_file
+
+        return self.get(identifier).read()

```

Comment:
If identifier refers an unknown file, the `get` method will return `None`. This will result in a *NoneType has no Attribute read*  error. I think its better to use the indexing notation here, like either

```Pyhton
self.__getitem__(identifier)
# or
self[identifier]
```
That will yield a more meaningful error message if identifier does not exist